src/pile.sol

```diff
@@ -129,15 +129,21 @@ contract Pile is DSNote {
         // --- Fee Accumulation ---
     function drip(uint fee) public {
         uint48 rho = fees[fee].rho;
+        require(now >= rho);
         uint speed = fees[fee].speed;
+        if (speed == 0) {
```

Comment:
Does it breafk if we not have this check? It will be more easily verifiable this way.

Reply:
Currently, it would break. If the `fee` is not set for a loan. but I think I can avoid it.

Reply:
Does it produce bad values or does it divide by zero? I think a revert here is fine if it's not set.

Reply:
The admit method will get a new parameter `fee` afterwards `speed==0` won't happen anymore.

Reply:
I'd prefer to leave it as a separate method and add a call to the reception that does both at the same time.

Reply:
The reception is only called by the borrower. I think it makes sense to pass it in the admit method and the admit contract calls `pile.file` afterwards.